index.js

```diff
@@ -38,153 +39,179 @@ module.exports.s3 = function(config, key, args, callback) {
 	);
 };
 
+var getDimArray = function( dims ) {
+	return ( typeof dims === 'string'
+		? dims.split(',')
+		: dims
+	).map(function(v) {
```

Comment:
Oof, can we split this out to two statements instead? `var dimArr = ... ? ... : ... ;`